staging/src/k8s.io/controller-manager/pkg/apis/genericcontrollermanager/install/install.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package install installs the experimental API group, making it available as
+// an option to all of the API encoding/decoding machinery.
+package install
+
+import (
+	"k8s.io/apimachinery/pkg/runtime"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/controller-manager/pkg/apis/genericcontrollermanager"
+	"k8s.io/controller-manager/pkg/apis/genericcontrollermanager/v1alpha1"
+	"k8s.io/kubernetes/pkg/api/legacyscheme"
+)
+
+func init() {
+	Install(legacyscheme.Scheme)
```

Comment:
Expose Scheme and Codecs variables here, and execute `AddToScheme` against the local Scheme variable